contracts/toCKB-typescript/src/switch/mod.rs

```diff
@@ -132,6 +136,32 @@ fn get_deletion_tx_type(data: &ToCKBCellDataView) -> Result<TxType, Error> {
     }
 }
 
+fn verify_xt(tx_type: &TxType) -> Result<(), Error> {
+    use TxType::*;
+    match tx_type {
+        MintXT
+        | PretermRedeem
+        | AttermRedeem
+        | AuctionSignerTimeout
+        | AuctionUnderCollateral
+        | AuctionFaultyWhenWarranty
+        | AuctionFaultyWhenRedeeming => Ok(()),
```

Comment:
```suggestion
        | PretermRedeem
        | AttermRedeem
        | AuctionSignerTimeout
        | AuctionFaultyWhenRedeeming => Ok(()),
```

for the xt token change ?

Reply:
yep